src/async_.rs

```diff
@@ -29,7 +29,7 @@ pub fn channel<T: Send>(size: usize) -> (Publisher<T>, Subscriber<T>) {
 }
 impl<T: Send> Publisher<T> {
     fn wake_all(&self) {
-        for sleeper in self.waker.sleepers.borrow().iter() {
```

Comment:
ReffCell removed around the sleepers vector of the Waker struct. It is no longer need to borrow a reference.